<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="ODA-L-011" failures="0" tests="2" name="GroupDemo" time="0.088" errors="0" timestamp="29 Jul 2018 05:28:48 GMT">
  <testcase classname="testclasses.CalculationTest" name="@AfterTest TearDown" time="0.001">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [7] but found [6],
	expected [8] but found [6]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [7] but found [6],
expected [8] but found [6]
at testclasses.CalculationTest.TearDown(CalculationTest.java:68)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- @AfterTest TearDown -->
  <testcase classname="testclasses.CalculationTest" name="TestSum" time="0.012"/>
  <testcase classname="testclasses.CalculationTest" name="TestSumBig" time="0.002"/>
  <testcase classname="testclasses.CalculationTest" name="TestSubstract" time="0.0">
    <ignored/>
  </testcase> <!-- TestSubstract -->
  <testcase classname="testclasses.CalculationTest" name="TestMultiply" time="0.0">
    <ignored/>
  </testcase> <!-- TestMultiply -->
  <testcase classname="testclasses.CalculationTest" name="TestDevision" time="0.0">
    <ignored/>
  </testcase> <!-- TestDevision -->
</testsuite> <!-- GroupDemo -->
